README.md

```diff
@@ -6,3 +6,69 @@ Atul Deshpande, Li-Fang Chu, Ron Stewart, Anthony Gitter.
 *bioRxiv* 2019. doi:10.1101/534834
 
 See the main [SINGE repository](https://github.com/gitter-lab/SINGE) for the software.
+
```

Comment:
Let's start a new `##` section here about the network inference methods.  Then each of the `##` sections below become `###` and the level 3 becomes level 4.

Before this section, we can add another `##` section describing the repository contents.  This can be a list like
- Commands for running network inference algorithms
- `dyngen` <describe subdir contents>
- `scripts` <describe subdir contents>